package.json

```diff
@@ -87,6 +87,7 @@
     "lodash": "4.17.15",
     "lottie-react-native": "^3.2.1",
     "lru-cache": "^4.1.3",
+    "moment": "^2.24.0",
```

Comment:
can we avoid usage of moment? this library tend to create performance issues.

like for tezos, we can probably use date-fns (e.g. `import differenceInCalendarDays from "date-fns/difference_in_calendar_days";`)